moveit_ros/move_group/default_capabilities_plugin_description.xml

```diff
@@ -65,5 +65,11 @@
       Provide a ROS service that allows for clearing the octomap
     </description>
   </class>
+  
+  <class name="move_group/MoveGroupTfPublisher" type="move_group::MoveGroupTfPublisher" base_class_type="move_group::MoveGroupCapability">
```

Comment:
no need for the "MoveGroup" prefix anywhere. we stopped using the prefix with newly added capabilities.